src/Services/privateApiService.js

```diff
@@ -1,21 +1,33 @@
-import axios from 'axios';
+import axios from "axios";
 
 const config = {
-    headers: {
-        Group: 91                //Aqui va el ID del equipo!!
-    }
-}
+  headers: {
+    Group: 91, //Aqui va el ID del equipo!!
+  },
+};
 
 const Get = () => {
-    axios.get('https://jsonplaceholder.typicode.com/users', config)
-    .then(res => console.log(res))
-    .catch(err => console.log(err))
-}
+  axios
+    .get("https://jsonplaceholder.typicode.com/users", config)
+    .then((res) => console.log(res))
+    .catch((err) => console.log(err));
+};
 
 export default Get;
 
 export const getAuthorizationHeader = () => {
-    const token = localStorage.getItem('token');
-    if(!token) return;
-    return {Authorization: `Bearer: ${token}`};
-}
+  const token = localStorage.getItem("token");
+  if (!token) return;
+  return { Authorization: `Bearer: ${token}` };
+};
+
+export const postAutorizationHeader = async (url, data) => {
+  const token = getAuthorizationHeader();
```

Comment:
the name should be only post or privatePost

Reply:
maybe authorizationHeader its a better name for this constant because its not a token, its an authorization header

Reply:
changes added